Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs

```diff
@@ -0,0 +1,284 @@
+            return response.Results[0].ResourceName;
+        }
+
+        /// <summary>
+        /// Creates a new Things to do campaign in the specified customer account.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="budget">The resource name of the budget for the new campaign.</param>
+        /// <param name="thingsToDoCenterAccountId">The Things to Do Center account ID.</param>
+        /// <returns>The resource name of the newly created campaign.</returns>
+        private static string CreateThingsToDoCampaign(GoogleAdsClient client, long customerId,
+            string budget, long thingsToDoCenterAccountId)
+        {
+
+            // Get the CampaignService.
+            CampaignServiceClient campaignService = client.GetService(Services.V14.CampaignService);
+
+            // [START add_things_to_do_ad_1]
+            // Creates a campaign.
+            Campaign campaign = new Campaign()
+            {
+                Name = "Interplanetary Cruise #" + ExampleUtilities.GetRandomString(),
+                AdvertisingChannelType = AdvertisingChannelType.Travel,
+                AdvertisingChannelSubType = AdvertisingChannelSubType.TravelActivities,
+
+                TravelCampaignSettings = new TravelCampaignSettings()
+                {
+                    TravelAccountId = thingsToDoCenterAccountId
+                },
+
+                // Recommendation: Set the campaign to PAUSED when creating it to prevent
+                // the ads from immediately serving. Set to ENABLED once you've added
+                // targeting and the ads are ready to serve
+                Status = CampaignStatus.Paused,
+
+                // Set the bidding strategy and budget.
+                MaximizeConversionValue = new MaximizeConversionValue(),
+                CampaignBudget = budget,
+
+                // Set the campaign network options.
+                NetworkSettings = new NetworkSettings
+                {
+                    TargetGoogleSearch = true
+                },
+
+                // Optional: Set the start date.
+                StartDate = DateTime.Now.AddDays(1).ToString("yyyyMMdd"),
+
+                // Optional: Set the end date.
+                EndDate = DateTime.Now.AddYears(1).ToString("yyyyMMdd"),
+            };
+            // [END add_things_to_do_ad_1]
+
+            MutateCampaignsResponse response = campaignService.MutateCampaigns(
+                customerId.ToString(), new CampaignOperation[] { new CampaignOperation() {
+                    Create = campaign
+                }}
+            );
+
+            string campaignResourceName = response.Results[0].ResourceName;
+            Console.WriteLine("Campaign with resource ID = '{0}' was added.", campaignResourceName);
```

Comment:
should be "...with resource name = ...."

Reply:
Done.